documentation/enforce/rules/lang/is_numeric/README.md

```diff
@@ -0,0 +1,34 @@
+# Lang | isNumeric
```

Comment:
IMO it is a `content` rule, as it doesn't check the type, but instead it parses the value.